ISX/ISX/Cells/ProgressBarCell.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  ProgressBarCell.swift
+//  ISX
+//
+//  Created by Jasper Zwiers on 21-11-17.
+//  Copyright © 2017 Maren Osnabrug. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class ProgressBarCell: UITableViewCell {
+    
+    @IBOutlet weak var targetRevenueTextLabel: UILabel!
+    @IBOutlet weak var progressBar: UIProgressView!
+    @IBOutlet weak var targetRevenueAmountLabel: UILabel!
+    
+    func setProgressBar (flight: Flight, currentRevenue: Int) {
+        var progress: Float
+        if(currentRevenue <= flight.averageRevenue) {
+            progress = (Float(currentRevenue)/Float(flight.highestRevenue))
+            targetRevenueTextLabel.text = "Target Revenue: €\(currentRevenue)/€\(flight.averageRevenue)"
+            targetRevenueAmountLabel.text = "€\(flight.averageRevenue)"
+        } else {
+            progress = (Float(currentRevenue)/Float(flight.highestRevenue))
```

Comment:
this line and the one below are the same in both statements, so they can be place above the if/else